app/helpers/sidebar_helper.rb

```diff
@@ -8,7 +8,7 @@ def cases_index_title
   def inbox_label
     unread_count = current_user.notifications.unread.count
     return "Inbox" if unread_count == 0
-    "Inbox <span class='badge badge-danger'>#{unread_count}</span>".html_safe
+    "Inbox <span class='badge bg-danger'>#{unread_count}</span>".html_safe
```

Comment:
Not using here?

Reply:
I checked all the other instances on the browser to confirm they still kept the same design characteristics, but this function is not being called anywhere on the app to confirm this one.

Reply:
Okay. I didn't notice this wasn't an html template either. All good.